app/index.js

```diff
@@ -395,7 +395,8 @@ app.on('ready', () => installDevExtensions(isDev).then(() => {
       createWindow,
       updatePlugins: () => {
         plugins.updatePlugins({force: true});
-      }
+      },
+      getPlugins: plugins.getModules
```

Comment:
Why do you rename it from `getModules` to `getPlugins` 🤔  I think it's better to have the same name so it's easier to read the code :)